changelogs/changelog-0.6.4.md

```
@@ -1,4 +1,5 @@
 # Major Changes
+# Major Changes
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)